HDF5Examples/C/TUTR/testh5cc.sh.in

```diff
@@ -523,17 +526,32 @@ elif [ -n "$H5_USE_114_API_DEFAULT" ]; then
     TOOLTEST -DH5_USE_110_API_DEFAULT $v110main
     TOOLTEST -DH5_USE_112_API_DEFAULT $v112main
     TOOLTEST $v114main
+elif [ -n "$H5_USE_116_API_DEFAULT" ]; then
+    echo "Testing HDF5  with 118_API_DEFAULT"
+    TOOLTEST -DH5_USE_16_API_DEFAULT $v16main
+    TOOLTEST -DH5_USE_18_API_DEFAULT $v18main
+    TOOLTEST -DH5_USE_110_API_DEFAULT $v110main
+    TOOLTEST -DH5_USE_112_API_DEFAULT $v112main
+    TOOLTEST -DH5_USE_114_API_DEFAULT $v114main
+    TOOLTEST -DH5_USE_116_API_DEFAULT $v114main
```

Comment:
The above sections only run TOOLTEST up to API_VERSION  - 2, so this line looks like it should be removed, though I don't know if that really affects anything